### ISSUE TITLE
Modify a process instance by activating elements inside a non-existing flow scope

### ISSUE THREAD

**Issue by saig0** [camunda/zeebe#9643]
## Description

We want to allow the activation of nested elements where the flow scope is not active yet. The element could be embedded in an embedded or an event subprocess. Before the given elements can be activated, their flow scope (or nested flow scope) must be activated first. The activation of a flow scope includes writing the events `activating` and `activated`, and creating the event subscriptions of the flow scope. After the flow scopes of a given element are activated, the element should be activated by writing an `activate` command. As a result, the process instance should continue at the given elements.

For example:
![image](https://user-images.githubusercontent.com/4305769/176389391-4e88feb9-7ea7-47ba-a328-33138acf9f4f.png)

* for each activate instruction:
  * write an `activating` and `activated` event for each flow scope (recursively) that is not active yet
  * create the event subscriptions of the activated flow scopes
  * write an `activate` command for the given element
* verify that 
  * the process instance activates the given elements and their flow scopes
  * the process instance activates multiple elements in the same flow scope
  * the process instance activates nested flow scopes of an element 
  * the process instance can complete the given elements, including the flow scopes
  * the process instance is completed when the given elements, the new flow scope instances, and the other element instances in this scope are completed

Out of scope:
* activate elements inside a given flow scope
* not supported elements (see #9729)
 
Blocked by
- #9642
